modules/pgamsspBidAdapter.js

```diff
@@ -0,0 +1,212 @@
+import { isFn, deepAccess, logMessage, logError } from '../src/utils.js';
+import { convertOrtbRequestToProprietaryNative } from '../src/native.js';
+
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+
+const BIDDER_CODE = 'pgamssp';
```

Comment:
Your bidder docs mention being tcfeu supported, we will need your GVL ID added to the adapter

Reply:
we updated doc